tools/project-context.js

```diff
@@ -1211,7 +1211,7 @@ _.extend(exports.CordovaPluginsFile.prototype, {
       // We just do a standard split here, not utils.parsePackageConstraint,
       // since cordova plugins don't necessary obey the same naming conventions
       // as Meteor packages.
-      var parts = line.split('@');
+      var parts = [line.substring(0,line.lastIndexOf('@')), line.substring(line.lastIndexOf('@')+1)];
```

Comment:
Since the complexity is increasing and there's duplication, perhaps a simple `cordova/package-version-parser.js` should be created?  And then something like:

```js
const { id, version } = require('./cordova/package-version-parser.js').parse(parts);
```

Just a thought. 💭